backend/kernelCI_app/views/treeDetailsSlowView.py

```diff
@@ -89,6 +91,11 @@ def __processFilters(self, request):
         except InvalidComparisonOP as e:
             return HttpResponseBadRequest(getErrorResponseBody(str(e)))
 
+    def __handle_test_environment_compatible(self, test_environment_compatible):
+        if test_environment_compatible is None:
+            return
+        return ",".join(test_environment_compatible).split(",")
```

Comment:
I'm puzzled by how this is working since the data in the database is a text, but you are never parsing that text, do you know how?

Reply:
The `environment_compatible` column returns an array like this `["google,veyron-jaq-rev5", "google,veyron-jaq-rev4"]`. So I use `join` to concatenate all in a string `"google,veyron-jaq-rev5,google,veyron-jaq-rev4"`, then use `split` to get the words separably `["google", "veyron-jaq-rev5", "google", "veyron-jaq-rev4"]`.

The main reason for this is to only use one for when I set the counter values

Reply:
oh, I'm using dbeaver and it shows the column type as `text`, it probably doesn't support postgres array